externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpDialect.h

```diff
@@ -0,0 +1,9 @@
+#ifndef TORCH_MLIR_DIALECTS_DIALECT_TCP_IR_TCPDIALECT_H_
+#define TORCH_MLIR_DIALECTS_DIALECT_TCP_IR_TCPDIALECT_H_
+
+#include "mlir/IR/Dialect.h"
+#include "mlir/IR/OpDefinition.h"
+
+#include "torch-mlir-dialects/Dialect/Tcp/IR/TcpDialect.h.inc"
+
+#endif // TORCH_MLIR_DIALECTS_DIALECT_TCP_IR_TCPDIALECT_H_
```

Comment:
Nit: Github flags these files as not ending with a newline, not sure if that's a problem.

Reply:
I'm not sure either. Added newline in all files anyways.